packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -28,6 +28,18 @@ export const PRODUCT_JETPACK_VIDEOPRESS_MONTHLY = 'jetpack_videopress_monthly';
 export const PRODUCT_JETPACK_SOCIAL_BASIC = 'jetpack_social_basic_yearly';
 export const PRODUCT_JETPACK_SOCIAL_BASIC_MONTHLY = 'jetpack_social_basic_monthly';
 
+//add-on products
+export const PRODUCT_JETPACK_BACKUP_ADDON_10GB_MONTHLY =
```

Comment:
These constant names are already long, but for consistency, I think it would be useful to make them match the string they represent; e.g., `PRODUCT_JETPACK_BACKUP_ADDON_STORAGE_10GB_MONTHLY`.

But since code is not my primary responsibility anymore, I defer to the feelings of the team. 🙂

Reply:
Thanks :) I have updated the name accordingly.